go/vt/vtgate/vindexes/vschema.go

```diff
@@ -97,10 +98,17 @@ func (ks *KeyspaceSchema) MarshalJSON() ([]byte, error) {
 		Sharded  bool              `json:"sharded,omitempty"`
 		Tables   map[string]*Table `json:"tables,omitempty"`
 		Vindexes map[string]Vindex `json:"vindexes,omitempty"`
+		Error    string            `json:"error,omitempty"`
 	}{
 		Sharded:  ks.Keyspace.Sharded,
 		Tables:   ks.Tables,
 		Vindexes: ks.Vindexes,
+		Error: func(ks *KeyspaceSchema) string {
```

Comment:
I think this is ok here, but should be used with care because inline functions incur additional overhead.